### PR TITLE
16702 download business summary from Administrative BN

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/16702

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by seeker25 — APPROVED** [review]
Please fix sonar cloud error if possible, also is there something in here that could replace that icon? https://pictogrammers.github.io/@mdi/font/6.5.95/ (I think we were using this for business-edit-ui)

Example:
https://github.com/bcgov/sbc-auth/blob/02798d2308eb6f757552526ea84acc014b4b2b91/auth-web/src/components/auth/account-freeze/PaymentReview.vue#L50

If not.. no worries

File: auth-web/src/components/auth/staff/admin/AdministrativeBN.vue [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-          </v-col>
-          <v-col col="6" class="pr-0" align="right">
-            <v-btn link color="primary" v-if="businessDetails" @click="resetSearch()">Search another Business</v-btn>
-            <v-btn link class="ml-2" v-if="businessDetails" @click="reload()">Reload</v-btn>
-          </v-col>
-        </v-row>
+        <v-col col="6">
+          <h2 class="view-header__title">Administrative BN</h2>
+        </v-col>
+        <v-col col="6" class="pr-0" align="right">
```
Comment (seeker25) [inline auth-web/src/components/auth/staff/admin/AdministrativeBN.vue:8]: The spacing problem is annoying, I'm having the same issue with this project - hoping to get some eslint rules so formatting is easier to manage.

**Comment by vysakh-menon-aot** [comment]
> ~Please fix sonar cloud error if possible~ bug is unrelated
> 
> also is there something in here that could replace that icon? https://pictogrammers.github.io/@mdi/font/6.5.95/ (I think we were using this for business-edit-ui)
> 
> Example:
> 
> https://github.com/bcgov/sbc-auth/blob/02798d2308eb6f757552526ea84acc014b4b2b91/auth-web/src/components/auth/account-freeze/PaymentReview.vue#L50
> 
> If not.. no worries - merge away

I did not find an icon for pdf download in mdi except a generic download icon

**Comment by vysakh-menon-aot** [comment]
@seeker25 Can you merge this pr? I don't have permission

**Pull request merged**